diff --git a/node_modules/terminal-kit/lib/detectTerminal.js b/node_modules/terminal-kit/lib/detectTerminal.js
index 263f57e..0af54b6 100644
--- a/node_modules/terminal-kit/lib/detectTerminal.js
+++ b/node_modules/terminal-kit/lib/detectTerminal.js
@@ -187,6 +187,11 @@ exports.guessTerminal = function( unpipe ) {
 			break ;
 	}
 
+  if(process.env.TRUECOLOR) {
+    appId = generic = 'xterm-truecolor' ;
+    safe=true;
+  }
+
 	return {
 		isTTY , isSSH , appId , safe , generic: safe ? appId : generic
 	} ;
